### ISSUE TITLE
[IMPROVEMENT] Indicator for a change in Chapter or Novel

### ISSUE THREAD

**Issue by uyennhiquang** [uyennhiquang/knk-novel-trial#15]
This might be more of a thing when visuals are incorporated, but suppose for a lowly text-only version as of now, is there an easy way to indicate that the player is about to turn to a new chapter or novel?

I think it may be nice to have at least an indicator that tells you what novel you're reading. Specifically, let's talk about when the player is about to start the very first novel. For all of the novel entries there is a prologue. So then, should the indicator come before or after the prologue? For the sake of the very first novel (and again, maybe even in general), I think there's something nice about a cold opening. Let's settle with an indicator after the prologue.

Imagine you're the player and the last sentence of the prologue is shown to you. You then click, not knowing that you're about to start the very first chapter. Instead of the first sentence of the chapter, clicking the window would instead display the title of the novel - starts and ends with a fading transition. During this transition, the player should not be able to continue until the transition is over. Once the transition is done, the game should display the first sentence of the chapter.
